site/src/pages/TemplatesPage/TemplatesPageView.tsx

```diff
@@ -171,7 +174,17 @@ export const TemplatesPageView: FC<TemplatesPageViewProps> = ({
     <Margins>
       <PageHeader
         actions={
-          canCreateTemplates && <CreateTemplateButton onNavigate={navigate} />
+          canCreateTemplates && (
+            <Button
+              startIcon={<AddIcon />}
+              variant="contained"
+              onClick={() => {
+                navigate("/starter-templates");
+              }}
```

Comment:
nit: Instead of using a button component that calls `navigate` you can use the button as a router link with the `href` property. 
```tsx
<Button as={LinkRouter} href="/starter-templates" />
```

Reply:
`RouterLink` expects a `to` prop, but `Button` only accepts a `href` prop, and the types don't work out. `Button` doesn't accept an `as` prop, just a `LinkComponent` prop.